src/pages/Settings.vue

```diff
@@ -0,0 +1,184 @@
+<template>
+  <PrimaryLayout title="HoloPort Settings">
+    <div class='settings'>
+      <h3 class='heading'>{{ deviceName }}</h3>
+      
+      <div class='settings-row'>
+        <div class='row-label'>
+          HPOS Version
+        </div>
+        <div class='row-value'>
+          {{ hposVersion }}
+        </div>
+      </div>
+
+      <div class='settings-row'>
+        <div class='row-label'>
+          <label for='device-name'>Device Name</label>
+        </div>
+        <div v-if='!isEditingDeviceName' class='row-value'>
+          {{ deviceName }}
+          <PencilIcon v-if='!isLoading' @click="editDeviceName" class='pencil' data-testid='edit-button' />
+        </div>
+        <div v-if='isEditingDeviceName' class='row-value' style="position: relative; top: -2px; margin-left: -2px">
+          <input v-model='editedDeviceName' class='device-input' id='device-name'>
+          <FilledCheckIcon @click="saveDeviceName" class='filled-check' data-testid='save-button' />
+          <CircledExIcon @click="cancelEditDeviceName" class='circled-ex' data-testid='cancel-button' />          
+        </div>
+      </div>
+
+      <div class='settings-row'>
+        <div class='row-label'>
+          Network
+        </div>
+        <div class='row-value'>
+          {{ networkStatus }}
+        </div>
+      </div>
+
+      <div class='settings-row'>
+        <div class='row-label'>
+          <label for='sshAccess'>Access for HoloPort support (SSH)</label>
+        </div>
+        <div class='row-value'>
+          <input type='checkbox' id='sshAccess' v-model='sshAccess'>
+        </div>
+      </div>
```

Comment:
If a host enables this how do they know what address to ssh into? Should that be displayed on this page as well?

Reply:
I think the ssh option is mostly there for tech support access, so tech support will know the ssh address. But it's a good question. That would be a product question, for @SirRobLyon, @marycamacho and @celestialhanley

Reply:
SSH access is there for tech support currently, not for hosts to manage their holoports.

The UX for host holoport management is via host console UI itself, not terminal.